src/components/DeferredInput.vue

```diff
@@ -1,19 +1,22 @@
 <script lang="ts">
 import debounce from 'lodash.debounce';
-import { Component, Prop, Vue, Watch } from 'vue-property-decorator';
+import { Component, Model, Prop, Vue, Watch } from 'vue-property-decorator';
 
 @Component({})
 export default class DeferredInput extends Vue {
+    @Model('change')
+    readonly value!: string;
+
     @Prop({type: Number, required: false, default: 250})
     readonly delay!: number;
 
-    internalValue = '';
+    internalValue = this.value;
 
     @Watch('internalValue')
     onInternalValueChange = debounce(this.emitChange, this.delay);
 
     emitChange() {
-        this.$emit('changed', this.internalValue);
+        this.$emit('change', this.internalValue);
     }
```

Comment:
Is this actually needed anymore now that the model binding is used?

Reply:
Yes, it's still needed to convey the changes to the `internalValue` to the external `value`.